sphinx_gherkindoc/utils.py

```diff
@@ -146,6 +151,7 @@ def display_name(path: pathlib.Path, package_name: Optional[str] = "") -> str:
     Args:
         path: Path for searching
         package_name: Sphinx-style, dot-delimited package name (optional)
+        display_name_from_dir: The
```

Comment:
`The`?? I think there might be more text here that is missing??

Reply:
Wanted to leave it to your imagination! 😄 WF